common/battery_monitor.h

```diff
@@ -97,6 +97,7 @@ BatteryMonitor() : reader_(batteryLevelPin,
       float v = battery();
       STDOUT.println(v);
 #ifdef ENABLE_AUDIO
+      talkie.Say(spBATTERYLEVEL);
```

Comment:
Some of these changes will use up some memory for everybody, and it has very little utility, so I don't think it's a good idea.

Maybe change this PR to only have the changes in your prop file and then I can merge it.

Reply:
Sure. I just edited the file back to original state instead of rebasing etc....